### PR TITLE
Minor update of the ArangoDB subcommand.

### PR DESCRIPTION
**Pull request by neunhoef**
I bumped the version to 0.2.1.
This is basically only a cleanup of the configuration options. In particular, I have added configuration options for
  - authentication with the mesos master
  - two switches for asynchronous replicas

### CODE REVIEW

**Comment by neunhoef** [comment]
I have no access to the results of the TeamCity build tests. Could somebody please let me know what is wrong, since it seems to be triggered by my changes?

File: repo/packages/A/arangodb/1/command.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "pip": [
+    "git+https://github.com/arangodb/arangodb-dcos"
```
Comment (kensipe) [inline repo/packages/A/arangodb/1/command.json]: we have previously allowed this.. however internally we discussed this and we would prefer to see the link for sub command be for a specific version and not off master.   we don't want changes to the master to be automatically added in without a versioning of the dcos service.   the idea is to have an immutable version.

**Comment by neunhoef** [comment]
OK, no problem. What is the syntax then for a branch or tag say? Is there any documentation on this?
According to the messages, this does not seem to be the cause of the problem.

**Comment by neunhoef** [comment]
From my perspective this looks now good.

**Comment by kensipe** [comment]
lgtm... merging

**Pull request merged**